plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClient.java

```diff
@@ -185,6 +187,7 @@
         extends BaseJdbcClient
 {
     private static final String ROWKEY = "ROWKEY";
+    private static final long MAX_LIMIT = 2000000;
```

Comment:
nit: Rename `MAX_TOPN_LIMIT`. This doesn't apply to plain `LIMIT` if I am understanding correctly.

Reply:
Also a code comment about the specific value would be nice. Why 2million instead of 1million or INT_MAX etc.

Reply:
Agreed. Happy to do a separate commit for the limit fix. Same PR is fine?

Reply:
Same PR should be fine.

Reply:
Done